eskip/parser.go

```diff
@@ -306,6 +306,7 @@ func (eskiprcvr *eskipParserImpl) Parse(eskiplex eskipLexer) int {
 	var eskiplval eskipSymType
 	var eskipVAL eskipSymType
 	var eskipDollar []eskipSymType
+	_ = eskipDollar // silence set and not used
```

Comment:
probably a new version of goyacc.
(reminder for others: this file is generated, and only stored in the repo for not breaking go get. maybe there is already a better way to deal with this, I just don't know yet.)